.eslintrc

```diff
@@ -15,6 +15,7 @@
     "block-scoped-var": "error",
     "camelcase": "error",
     "comma-dangle": ["error", "always-multiline"],
+    "consistent-this": ["error", "self"],
```

Comment:
In most directive controllers we were using `this` to refer to the controller instance and only in three were we were using `vm`. I figure we should try to be consistent one way or the other, this commit fixes two of those to use `this` except for `annotation` where I felt it would need enough changes it was probably better left for a separate commit.